api/namex/models/request.py

```diff
@@ -111,6 +111,19 @@ class Request(db.Model):
     REQUEST_FURNISHED = 'Y'
 
     # properties
+    @property
+    def has_consumed_name(self):
+        names = self.names.all()
+
+        def is_not_consumed(name):
```

Comment:
same thing, should this be in the names model?

Reply:
Same as my comment above.